aggregator/aggregator.go

```diff
@@ -148,7 +148,8 @@ type aggregator struct {
 	instanceID    string
 	shardFn       ShardFn
 	checkInterval time.Duration
-	handler       Handler
+	flushManager  FlushManager
```

Comment:
it's a bit weird that aggregator keeps ref to flushMgr and flushHandler just to close them, maybe they should be closed in main() or serve()?